### ISSUE TITLE
Trying to get it to work in ionic 3

### ISSUE THREAD

**Issue by Haroon6138** [mmig/cordova-plugin-gadgetbridge#3]
Hey, Thanks a lot for this awesome plugin, Basically i am trying to get this plugin to work with ionic 3, i have successfully added this plugin into my project, but i am unable to call any of the methods like openView etc. etc. i keep getting error: 

`ERROR ReferenceError: GadgetbridgePlugin is not defined`

Please can you help me or point me to the right direction ASAP as it's really urgent...

**Comment by russaa** [comment]
did you store the plugin instance into the variable `GadgetbridgePlugin` first?
i.e. something like
```javascript
var GadgetbridgePlugin = window.cordova.plugins.gadgetbridgePlugin;
```
 (it's not directly attached to `window`)

**Comment by Haroon6138** [comment]
yes i did, i even tried like this:

```
declare var cordova: any;

cordova.plugins.gadgetbridgePlugin.openView("MiBandPairingActivity", this.onSuccess, this.onFailure);
```
but still the same error

**Comment by russaa** [comment]
... and you are waiting until `cordova` has been initialized, before trying to access/use the plugin?
 e.g. using ionic's `Platform`
```javascript
platform.ready().then(() => {
  var GadgetbridgePlugin = window.cordova.plugins.gadgetbridgePlugin;
  ...
```

**Comment by Haroon6138** [comment]
yes i am waiting, infact i am calling this function on button click just to be on the safe side. when i am running the app on iOS device, i get the following error:

`ERROR: Plugin 'GadgetbridgePlugin' not found, or is not a CDVPlugin. Check your plugin mapping in config.xml.`

but i have double-checked my config.xml & my plugins list, the plugin is infact installed there. 

<img width="1786" alt="screen shot 1440-06-15 at 11 44 11 am" src="https://user-images.githubusercontent.com/47740408/53078280-e8991a80-3504-11e9-864d-b9447d63b805.png">

**Comment by russaa** [comment]
ah OK -- sorry for not stating this more prominently:  
this plugin is only for the `android` platform

under the hood it uses the awesome [Gadgetbrigde](https://github.com/Freeyourgadget/Gadgetbridge) app which, unfortunately, is only available for Android

(I'll make it more clear in the README)

**Comment by Haroon6138** [comment]
ahh okay got it. thanks a lot for your support....
